src/QualificationRunner/Program.cs

```diff
@@ -24,7 +25,9 @@ static int Main(string[] args)
             .WithParsed(startCommand)
             .WithNotParsed(err => _valid = false);
 
-         Console.ReadLine();
+         if (Debugger.IsAttached)
+            Console.ReadLine();

```

Comment:
@Yuri05 I want the readlin to happen in debug so that we can see what;s going on. Otherwise the view closes automatically